tests/bgp/test_bgp_update_timer.py

```diff
@@ -0,0 +1,351 @@
+"""Check how fast FRR or QUAGGA will send updates to neighbors."""
+import contextlib
+import ipaddress
+import logging
+import pytest
+import requests
+import tempfile
+import time
+
+from scapy.all import sniff, IP
+from scapy.contrib import bgp
+from tests.common.utilities import wait_tcp_connection
+
+
+pytestmark = [
+    pytest.mark.topology("any"),
+]
+
+BGP_SAVE_DEST_TMPL = "/tmp/bgp_%s.j2"
+NEIGHBOR_SAVE_DEST_TMPL = "/tmp/neighbor_%s.j2"
+BGP_LOG_TMPL = "/tmp/bgp%d.pcap"
+ANNOUNCED_SUBNETS = [
+    "10.10.100.0/27",
+    "10.10.100.32/27",
+    "10.10.100.64/27",
+    "10.10.100.96/27",
+    "10.10.100.128/27"
+]
+NEIGHBOR_ASN0 = 61000
+NEIGHBOR_ASN1 = 61001
+NEIGHBOR_PORT0 = 11000
+NEIGHBOR_PORT1 = 11001
+
+
+def _write_variable_from_j2_to_configdb(duthost, template_file, **kwargs):
+    save_dest_path = kwargs.pop("save_dest_path", "/tmp/temp.j2")
+    keep_dest_file = kwargs.pop("keep_dest_file", False)
+    duthost.host.options["variable_manager"].extra_vars.update(kwargs)
+    duthost.template(src=template_file, dest=save_dest_path)
```

Comment:
Can you use the jinja2.Template and duthost.copy module to generate json config file? This is more explicit than updating extra vars of duthost's variable_manager.

Example:
https://github.com/Azure/sonic-mgmt/blob/master/tests/bgp/test_bgp_bbr.py#L44

Reply:
Sure, will update.